### PR TITLE
fix/text-extractor-fix-workflow

### PR DESCRIPTION
**Pull request by kirtimanmishrazipstack**
## What

- Error handling for text-extractor on backend 

## Why

-  text-extractor was producing error for db connectors
- 

## How

- Changes on workflow_helper (on producing error log) and destination (to handle the fix for output by text-extractor)

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- NO

## Database Migrations

- N/A

## Env Config

- N/A

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

- Tested the workflow with the changes

## Screenshots

- 
Screenshot from 2024-09-02 11-13-52

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/workflow_manager/workflow/workflow_helper.py [hunk 1]
```diff
@@ -156,6 +156,8 @@ def process_input_files(
             except Exception as e:
                 failed_files += 1
                 error_message = f"Error processing file '{file_path}'. {e}"
+                logger.error(error_message)
+                logger.error(f"Error {traceback.format_exc()}")
```
Comment (ritwik-g) [inline backend/workflow_manager/workflow/workflow_helper.py:160]: @kirtimanmishrazipstack no need to separately log traceback like this. Follow what's been done below 

https://github.com/Zipstack/unstract/blob/main/worker%2Fsrc%2Funstract%2Fworker%2Fworker.py#L224-L229

Comment (kirtimanmishrazipstack) [reply backend/workflow_manager/workflow/workflow_helper.py]: done @ritwik-g

**Review by chandrasekharan-zipstack — APPROVED** [review]

File: backend/workflow_manager/endpoint/destination.py [hunk 2]
```diff
@@ -272,7 +272,9 @@ def insert_into_db(self, input_file_path: str) -> None:
         if not data:
             return
         # Remove metadata from result
-        data.pop("metadata", None)
+        # Tool text-extractor returns data in the form of string. Don't pop out metadata
```
Comment (hari-kuriakose) [inline backend/workflow_manager/endpoint/destination.py:275]:
```suggestion
        # Tool text-extractor returns data in the form of string. Don't pop out metadata in this case.
```

**Review by jaseemjaskp — APPROVED** [review]
LGTM

**Pull request merged**